<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookForm</class>
 <widget class="QWidget" name="BookForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>add</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>insert book informations</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>title :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>autor :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>edition :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="new_book_title"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="new_book_autor"/>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="new_book_edition">
             <property name="displayFormat">
              <string>yyyy/mm/dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>copies :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="new_book_copies">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>cancel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>confirm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="insert_status">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>database status</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>edit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>search book</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>by id :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>by title :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>insert new informations</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>id :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>title :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>autor :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>edition :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>copies :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="edited_book_title"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="edited_book_autor"/>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="edited_book_edition">
             <property name="displayFormat">
              <string>yyyy/mm/dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="edited_book_id"/>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="edited_book_copies">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>aivables :</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="edited_book_aivables">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="book_edit_confirm">
             <property name="text">
              <string>confirm</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="book_edit_cancel">
             <property name="text">
              <string>cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="edit_status">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>database status</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
